miden/Cargo.toml

```diff
@@ -52,6 +52,7 @@ serde_json = {version = "1.0.59", optional = true }
 stdlib = { package = "miden-stdlib", path = "../stdlib", version = "0.3", default-features = false }
 structopt = { version = "0.3", default-features = false, optional = true }
 verifier = { package = "miden-verifier", path = "../verifier", version = "0.4", default-features = false }
+vm-core = { package = "miden-core", path = "../core", version = "0.4", default-features = false }
```

Comment:
Why do we need this import here? Could we not get all needed info from `prover` or `processor` crates?

Reply:
Kernel wasn't exported there, but it indeed should. Fixed in `processor`!